lib/src/app/app.dart

```diff
@@ -729,4 +729,76 @@ class AppClient {
     final request = CreateKeyFromExistingKeyAuthorizationsRequest()..id = id;
     return await _client.createKeyFromExistingKeyAuthorizations(request);
   }
+
+  /// Retrieves user-defined [Metadata] for an organization.
+  ///
+  /// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
+  Future<GetOrganizationMetadataResponse> getOrganizationMetadata(String id) async {
+    final request = GetOrganizationMetadataRequest()..organizationId = id;
+    return await _client.getOrganizationMetadata(request);
+  }
+
+  /// Updates user-defined [Metadata] for an organization.
+  ///
+  /// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
+  Future<void> updateOrganizationMetadata(String id, Map<String, dynamic> data) async {
+    final request = UpdateOrganizationMetadataRequest()
+      ..organizationId = id
+      ..data = _mapToStruct(data);
```

Comment:
`_mapToStruct` is undefined, which is causing errors. I think what you're looking for is `data.toStruct()`.